virttest/qemu_vm.py

```diff
@@ -1420,6 +1421,44 @@ def sort_key(dev):
             if params.get("pci_controllers_autosort", "yes") == "yes":
                 pcics.sort(key=sort_key, reverse=False)
             devices.insert(pcics)
+
+        def select_iothread_manager(devices, params):
+            """Dispatch iothread manager."""
+            iothreads = params.get("iothreads")
+            iothread_scheme = params.get("iothread_scheme")
```

Comment:
please set 'oto' as default scheme.

Reply:
I believe it is better to set a default in kar.

Reply:
ok for me.